ios/RNTPTDocumentView.m

```diff
@@ -4654,6 +4670,21 @@ - (NSString *) getDocumentPath {
     return [resultMap copy];
```

Comment:
With my other suggestions, `resultMap` will no longer be the mutable version so you don't need to return the copy:

```suggestion
    return resultMap;
```